benchmarks/distributed/rpc/parameter_server/configurations/benchmark_configurations.json

```diff
@@ -4,5 +4,17 @@
         "ps_count": 0,
         "rpc_async_timeout": 15,
```

Comment:
is this timeout just for `rpc_async` and not for `rpc_sync` / `remote`? If so, any reason?

Reply:
I was using the timeout on the training calls that use rpc_async. I'll move it to the rpc_backend_options.